Assets/Scripts/Utilities/DebugUtils.cs

```diff
@@ -8,7 +8,7 @@ public static void Assert( bool condition, string message = "" )
 		if ( !condition )
 		{
 			Debug.LogError( message );
-			System.Diagnostics.Debugger.Break();
+			// Debug.Break(); // Uncomment if you want to pause the editor when an assert is hit.
 		}
```

Comment:
This function should have a bool like BreakOnAssert with default value false. Then Debug.Break only happens if that's true.